packages/common/core-interfaces/api-report/core-interfaces.legacy.alpha.api.md

```diff
@@ -271,6 +271,7 @@ export interface IFluidHandleInternal<out T = unknown> extends IFluidHandle<T>,
     readonly absolutePath: string;
     attachGraph(): void;
     bind(handle: IFluidHandleInternal): void;
+    readonly metadata?: Readonly<Record<string, number | boolean | string>> | undefined;
```

Comment:
any way to not leak this in the expose api?

Reply:
Oh no, I made the bad assumption that IFluidHandleInternal was internal :-P

Yeah I'll take a look.

Reply:
Done in latest